staging/src/k8s.io/kubectl/pkg/cmd/apply/apply.go

```diff
@@ -172,9 +172,11 @@ func NewCmdApply(baseName string, f cmdutil.Factory, ioStreams genericclioptions
 		Long:                  applyLong,
 		Example:               applyExample,
 		Run: func(cmd *cobra.Command, args []string) {
-			cmdutil.CheckErr(o.Complete(f, cmd))
+			o.RecordFlags.Complete(cmd)
```

Comment:
Do you need to CheckErr this?
```
cmdutil.CheckErr(o.RecordFlags.Complete(cmd))
```

I see in the implementation that it always returns nil, but someone could change it in the future to actually return an error.